### PR TITLE
PNDA-4547: Create flink symlink for yarn-wrapper.

### PR DESCRIPTION
**Pull request by Hitesh-Maplelabs**
Problem Statement:
Yarn queue wrapper is not working for flink. 

Changes Done:
Create Symlinks for flink options, which are being called by  yarn-wrapper.

Test Details:
AWS-RHEL-PICO

### CODE REVIEW

**Comment by stephanesan** [comment]
The Title, 'Changes Done' in the PR details and commit message do not match the resolution.

File: salt/flink/init.sls [hunk 1]
```diff
@@ -135,3 +138,12 @@ flink-copy_dependency_jar-{{ filename }}:
     - name: {{ flink_real_dir }}/lib/{{ filename }}
     - source: {{ file }}
 {% endfor %}
+
+{% for exec in execs %}
+flink_symlink_{{ exec }}:
+  file.symlink:
```
Comment (stephanesan) [inline salt/flink/init.sls:144]: In which situation is this symlink expected to be used (instead of the wrapped version prioritized in the user's PATH)?

Comment (Hitesh-Maplelabs) [reply salt/flink/init.sls:144]: This Symlink will be used by yarn-wrapper.

**Comment by Hitesh-Maplelabs** [comment]
Updated the 'Changes Done' and commit message.

File: salt/flink/init.sls (hunk 1 above)
Comment (stephanesan) [reply salt/flink/init.sls:144]: OK, no specific version needed then. Can the link point to the non-versioned 'flink_link_dir' (to simplify the upgrade process) instead?

Comment (Hitesh-Maplelabs) [reply salt/flink/init.sls:144]: Done.

**Pull request merged**